### PR TITLE
Added firstinitial.lastname, lastname_firstinitial, firstnamelastname…

### PR DESCRIPTION
**Pull request by Godmartinz**
… and firstnamelastinitial to username formats.

### CODE REVIEW

File: resources/lang/en-GB/general.php [hunk 1]
```diff
@@ -88,6 +88,9 @@
     'firstname_lastname_underscore_format' => 'First Name Last Name (jane_smith@example.com)',
     'lastnamefirstinitial_format' =>  'Last Name First Initial (smithj@example.com)',
     'first'					=> 'First',
+    'firstnamelastname'     => 'First Name Last Name (janesmith@example.com)',
+    'lastname_firstinitial' => 'Last Name First Initial (smith_f@example.com)',
```
Comment (snipe) [inline resources/lang/en-GB/general.php]: This should probably be `smith_j@example.com`, no?

File: resources/lang/en/general.php [hunk 2]
```diff
@@ -88,6 +88,11 @@
     'firstname_lastname_underscore_format' => 'First Name Last Name (jane_smith@example.com)',
     'lastnamefirstinitial_format' =>  'Last Name First Initial (smithj@example.com)',
     'first'					=> 'First',
+    'firstnamelastname'     => 'First Name Last Name (janesmith@example.com)',
+    'lastname_firstinitial' => 'Last Name First Initial (smith_f@example.com)',
```
Comment (snipe) [inline resources/lang/en/general.php]: Same here, since we're using Jane Smith as the dummy name. The `f` might be confusing for folks

**Comment by snipe** [comment]
This looks good - just two minor nits in the language file example addresses, and I think it's ready to go.

**Review by snipe — CHANGES_REQUESTED** [review]
Just those two small nits

File: resources/lang/en-GB/general.php (hunk 1 above)
Comment (Godmartinz) [reply resources/lang/en-GB/general.php]: typo has been corrected.

File: resources/lang/en/general.php (hunk 2 above)
Comment (Godmartinz) [reply resources/lang/en/general.php]: typo corrected

**Comment by snipe** [comment]
Nice, thanks!

**Pull request merged**